core/src/main/java/com/expedia/adaptivealerting/core/metric/Metric.java

```diff
@@ -0,0 +1,46 @@
+package com.expedia.adaptivealerting.core.metric;
+
+import com.expedia.www.haystack.commons.entities.MetricPoint;
+import scala.Enumeration;
+import scala.collection.JavaConverters;
+
+import java.util.Map;
+
+public class Metric {
+    private String name;
+    private String type;
+    private Map<String, String> tags;
+
+    public Metric() {
+    }
+
+    public Metric(MetricPoint metricPoint) {
```

Comment:
Remove this constructor per comments above. A util method on a MetricUtil class would be fine.